app/src/main/res/layout/upload.xml

```diff
@@ -9,101 +9,121 @@
         android:id="@+id/relativeLayout1"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:paddingLeft="10dp"
+        android:paddingRight="10dp"
         >
 
         <TextView
             android:id="@+id/textView1"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_alignParentLeft="true"
             android:layout_centerVertical="true"
-            android:layout_marginLeft="10dp"
             android:text="@string/accounts"
-            android:textAppearance="?android:attr/textAppearanceMedium"
-            android:textColor="@color/blue_light"
-            android:textStyle="bold" />
+            android:textSize="16sp"
```

Comment:
Please try to add all dimensions in the `dimens.xml` as much as possible. I know this has not been the case in the current codebase, but try to follow the conventions wherever possible.